### PR TITLE
clarified link to format in README.md

### PR DESCRIPTION
**Pull request by soerenthomsen**
as we will add more links here I suggest to be more specific

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ![ OceanGliders Logo](logo-ocean-gliders.png "OceanGliders Logo")
 
 # OceanGliders format-user-manual
-Here we review issues related to OceanGliders format and vocabularies.
+Here we review issues related to OceanGliders (OG) format and vocabularies.
 A discussion section is also available for any question or comment related to OG format and vocabularies.
 
-You can find the most recent version [here](https://github.com/OceanGlidersCommunity/OG-format-user-manual/blob/main/OG_Format.adoc)
+You can find the most recent version of the OG format [here](https://github.com/OceanGlidersCommunity/OG-format-user-manual/blob/main/OG_Format.adoc)
```
Comment (castelao) [inline README.md:7]: A suggestion. The 'here' doesn't add any information. Could be removed.

... of the [OG format](https://...

**Review by castelao — COMMENTED** [review]
A suggestion. Feel free to don't follow it.

**Comment by soerenthomsen** [comment]
Agreed and implemented. @justinbuck could you approve so we can merge?

**Review by castelao — APPROVED** [review]

**Comment by soerenthomsen** [comment]
@justinbuck do you have time to give it a quick review?

**Comment by castelao** [comment]
Or @glidermann ?

**Review by justinbuck — APPROVED** [review]
Changes look good and tested the link successfully

**Pull request merged**